cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,58 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+import NewArticlePageObject from '../support/pages/newArticle.pageObject';
+import ArticlePageObject from '../support/pages/article.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
 
-describe('Article', () => {
-  before(() => {
-
-  });
+const newArticlePage = new NewArticlePageObject();
+const articlePage = new ArticlePageObject();
+const homePage = new HomePageObject();
+let article;
 
+describe('Article', () => {
   beforeEach(() => {
     cy.task('db:clear');
+    cy.task('generateArticle')
+      .then(generateArticle => {
+        article = generateArticle;
+      });
   });
 
   it('should be created using New Article form', () => {
-
+    cy.register();
+    newArticlePage.visit();
+    newArticlePage.typeArticleTitle(article.title);
+    newArticlePage.typeArticleDescription(article.description);
+    newArticlePage.typeArticleBody(article.body);
+    newArticlePage.typeArticleTag(article.tag);
+    newArticlePage.clickSubmitBtn();
+    articlePage.assertArticleTitle(article.title);
+    articlePage.assertArticleBody(article.body);
   });
 
   it('should be edited using Edit button', () => {
-
+    newArticlePage.visit();
+    cy.reload();
```

Comment:
Avoid using `cy.reload()` without necessity. It can lead to longer test execution times and unpredictable behavior if the state is not properly reset.